Assets/Docs/Design/Initial_Startup_System_Design.md

```
@@ -189,22 +189,22 @@ public class SceneTransitionManager : Singleton<SceneTransitionManager>
 ### 2. 이벤트 흐름
 
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)